### PR TITLE
Bookinglist

### PR DESCRIPTION
**Pull request by malenesoeholm**

### CODE REVIEW

File: src/react/views/BookingList.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        text='Are you sure that you want to delete this booking?'/>
+    </View>
+  }
+
+  renderBooking (booking, index, sortedBookings) {
+    const bookingDate = booking.from.clone().startOf('day')
+    const prevDate = index < 1 ? bookingDate : sortedBookings[index - 1].from.clone().startOf('day')
+
+    return <View key={booking.id}>
+      {!index || prevDate.isBefore(bookingDate) ? this.renderSectionHeader(bookingDate) : null}
```
Comment (budde377) [inline src/react/views/BookingList.js]: No need to test for index here

File: src/react/views/LoggedInApp.js [hunk 2]
```diff
@@ -130,16 +143,29 @@ export default class LoggedInApp extends Backable {
...
+  }
+
   onPressSettings (navigator) {
     // Checking if the settings button has not already been clicked
     if (navigator.getCurrentRoutes().length < 2) {
       navigator.push(this.settingsRoute)
     }
   }
 
+  // TODO if the user has no bookings, then perhaps the button for Booking List should not be displayed?
```
Comment (budde377) [inline src/react/views/LoggedInApp.js]: You have fixed this?

File: src/stateHandler.js [hunk 3]
```diff
@@ -133,7 +133,7 @@ class StateHandler extends EventEmitter {
 
   refresh () {
     if (!this._auth) return this._setupAuth()
-    this._setupAuth(this._auth.userId, this._auth.token)
+    this._setupAuth(this._auth.rberrId, this._auth.token)
```
Comment (budde377) [inline src/stateHandler.js]: rberrId???

File: src/react/views/BookingList.js (hunk 1 above)
Comment (malenesoeholm) [inline src/react/views/BookingList.js]: If index is 0, then we also need to render the section header.

**Pull request merged**